modules/mpv-player/android/src/main/java/expo/modules/mpvplayer/PiPController.kt

```
@@ -106,11 +106,24 @@ class PiPController(private val context: Context, private val appContext: AppCon
+        // after the player has unmounted.
+        playbackRate = 0.0
         isInPiPMode = false
         pipEntryNotified = false
         unregisterLifecycleCallbacks()
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
             val activity = getActivity()
+            // Re-push params with auto-enter disabled so the system stops
+            // considering this task eligible for auto-PiP on home press.
+            try {
+                activity?.setPictureInPictureParams(buildPiPParams())
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)